pkgs/desktops/gnome-3/extensions/window-is-ready-remover/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "window-is-ready-remover";
+  version = "1.02";
+
+  src = fetchFromGitHub {
+    owner = "nunofarruca";
+    repo = "WindowIsReady_Remover";
+    rev = "a9f9b3a060a6ba8eec71332f39dc2569b6e93761";
```

Comment:
Could you ask upstream to add git tags for releases? It will be easier to update this in the future and also safer.

Reply:
Couldn't managed  it. But from [package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming) it should be ok.